packages/varlet-ui/src/date-time-picker/__tests__/index.spec.js

```diff
@@ -4,32 +4,51 @@ import { mount } from '@vue/test-utils'
 import DatePicker from '../../date-picker'
 import TimePicker from '../../time-picker'
 import { delay } from '../../utils/test'
+import { expect, vi } from 'vitest'
+
+const Wrapper = {
+  components: {
+    [DateTimePicker.name]: DateTimePicker,
+    [DatePicker.name]: DatePicker,
+    [TimePicker.name]: TimePicker,
+  },
+  props: ['onConfirm', 'onCancel', 'prevStepText', 'nextStepText', 'cancelButtonText', 'confirmButtonText'],
```

Comment:
All properties and events are tested together and not a good job. Please test this component separately.